File: cses Introductory/Repetitions/repetitions.cpp
#include<iostream>
#include<string>
using namespace std;
 
string s;
int rep = 1;
int absRep = 1;
 
int main(){
    cin >> s;
    for(int i = 0; i < s.size(); i++){
        if(s[i+1] == s[i]) rep++;
        else rep = 1;
        if(i == s.size()-1 && s[i] == s[i-1]) rep++;
        if(rep > absRep) absRep = rep;
    }
    cout << absRep;
}

File: cses Introductory/Repetitions/repetitions.py
s = input()
rep = 1
absRep = 1
for i in range(0, len(s)-1):
    if s[i] == s[i+1]:
        rep += 1
    else:
        rep = 1
    if rep > absRep:
        absRep = rep
print(absRep)